### PR TITLE
RIFEX-51 Pay to RNS domain

### PR DESCRIPTION
**Pull request by JONAF2103**
This PR introduces the pay to rns ui and service. We can pay into a rns domain now using the ui.

### CODE REVIEW

File: app/scripts/background.js [hunk 1]
```diff
@@ -28,6 +28,9 @@ const setupMetamaskMeshMetrics = require('./lib/setupMetamaskMeshMetrics')
 const EdgeEncryptor = require('./edge-encryptor')
 const getFirstPreferredLangCode = require('./lib/get-first-preferred-lang-code')
 const getObjStructure = require('./lib/getObjStructure')
+const regeneratorRuntime = require('regenerator-runtime');
```
Comment (marcosmartinez7) [inline app/scripts/background.js:31]: Why is this needed?

File: package.json [hunk 2]
```diff
@@ -213,16 +216,18 @@
     "valid-url": "^1.0.9",
     "vreme": "^3.0.2",
     "web3": "^0.20.7",
-    "web3-utils": "^1.2.4",
     "web3-eth-abi": "^1.2.6",
     "web3-stream-provider": "^4.0.0",
-    "xtend": "^4.0.1"
+    "web3-utils": "^1.2.4",
+    "xtend": "^4.0.1",
+    "@rsksmart/lumino-light-client-sdk": "file:../lumino-light-client-sdk"
```
Comment (marcosmartinez7) [inline package.json:223]: i think we should not commit this line since we are under development

File: app/scripts/background.js (hunk 1 above)
Comment (JONAF2103) [reply app/scripts/background.js:31]: Because of the lumino-light-client-sdk, it uses it when it gets compiled to transform async await operations, so basically if we don't define that in the background script we don't have it in the compiled version and when it's called it fails because it's undefined.

File: package.json (hunk 2 above)
Comment (JONAF2103) [reply package.json:223]: I think it doesn't heart anyone since we only need to change that later to a specific version, but if you want i can remove it. I mean it's just a help to the developer, if we remove it we need to specify somewhere that we need to link it, and npm link is really old at this point, file:.. syntax is better and more clear i think.

Comment (marcosmartinez7) [reply package.json:223]: Youre right!

**Review by rorolopetegui — COMMENTED** [review]
I agree with the change of the "link" to the SDK, but as you say, npm link is a little bit older right now. 
Lgtm in general

**Comment by marcosmartinez7** [comment]
lgtm for now! 

i know is a draft for now, but change the console.log for debug

**Comment by JONAF2103** [comment]
This PR is not longer valid since 
https://github.com/anarancio/nifty-wallet/pulls has this and more updated content, i'm closing it.
